RevenueCatUI/Templates/V2/Components/Text/TextComponentView.swift

```diff
@@ -120,13 +120,53 @@ private struct NonLocalizedMarkdownText: View {
             }
         }
 
+        // Handle underline with <u>text</u> syntax
+        attrString = Self.applyUnderlines(to: attrString)
+
         return attrString
 
         #else
         return nil
         #endif
     }
 
+    /// Processes `<u>text</u>` syntax and applies underline styling
+    private static func applyUnderlines(to attrString: AttributedString) -> AttributedString {
+        var result = attrString
+        let plainString = String(result.characters)
+
+        // Find all <u>...</u> matches
+        guard let regex = try? NSRegularExpression(pattern: "<u>(.*?)</u>", options: []) else {
```

Comment:
Also maybe silly, but just to clarify. This allows `<u></u>` right? Is that intentional?

Reply:
didn't think about that case honestly, but I'm not concerned